packages/typespec-vscode/src/tsp-language-client.ts

```diff
@@ -0,0 +1,113 @@
+import { ExtensionContext, LogOutputChannel, workspace } from "vscode";
```

Comment:
Refactor the language client code to a separated code as a class and fix the restart bug in the meantime